config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -56,6 +56,10 @@ presubmits:
         - "--scenario=execute"
         - "--"
         - "./scripts/ci-test.sh"
+        resources:
+          requests:
+            cpu: 6
```

Comment:
Umm ... I'm going to defer to @kubernetes/test-infra-maintainers here.  This seems like alot for a pre-submit.

Reply:
Scaling back the request a little after talking with @BenTheElder

Reply:
For the sake of context: we currently run tests on a cluster with 800 cores and 5200 GB of RAM, and run maybe 300 tests simultaneously.

Assuming that these tests are actually reasonably CPU bound for their duration (i.e. the request is not spurious and we aren't just spending most of our time waiting on the network), the new numbers seem pretty reasonable to me.

Reply:
worth noting these are _requests_ and test-infra sets no limits currently 🤷‍♂ 
however, the values were entirely cargo culted initially, and I don't think they're actually terribly CPU bound because they're integration (~e2e) testing with a kind cluster, that generally is quite cheap and involves lots of suspended code waiting for things to happen.